concourse/pipelines/artifact-releaser-autopush.jsonnet

```diff
@@ -0,0 +1,133 @@
+local imgpublishjob = {
+  local job = self,
+
```

Comment:
It's probably best to move this migration to its own PR  -- I think it's a good practice to have a single PR cover a single issue. "Insert image task" seems like a nice atomic change, and "convert artifact releaser to jsonnet" seems like another nice atomic change.

Reply:
Thanks for the review, I forgot to delete this PR after Liam's initial comment but thanks for the pointers for transitioning to jsonnet! I'll separate these out into separate PRs.